simulator-ui/src/main/webapp/app/entities/scenario-execution/list/scenario-execution.component.html

```diff
@@ -65,76 +65,79 @@ <h2 id="page-heading" data-cy="ScenarioExecutionHeading" *ngIf="!hideTitle">
-              scenarioExecution.status
-            }}</span>
-          </td>
-          <td>{{ scenarioExecution.errorMessage }}</td>
-          <td class="text-end">
-            <div class="btn-group">
-              <button
-                type="button"
-                [routerLink]="['/scenario-action']"
-                [queryParams]="{ 'filter[scenarioExecutionId.in]': scenarioExecution.executionId }"
-                class="btn btn-info btn-sm"
-                data-cy="filterOtherEntityButton"
-              >
-                <fa-icon icon="flag"></fa-icon>
-                <span
-                  class="d-none d-md-inline"
-                  jhiTranslate="entity.action.show"
-                  [translateValues]="{ otherEntity: ('citrusSimulatorApp.scenarioExecution.scenarioActions' | translate) }"
-                  >Show Actions</span
-                >
-              </button>
-              <button
-                type="button"
-                [routerLink]="['/message']"
-                [queryParams]="{ 'filter[scenarioExecutionId.in]': scenarioExecution.executionId }"
-                class="btn btn-info btn-sm"
-                data-cy="filterOtherEntityButton"
-              >
-                <fa-icon icon="message"></fa-icon>
-                <span
-                  class="d-none d-md-inline"
-                  jhiTranslate="entity.action.show"
-                  [translateValues]="{ otherEntity: ('citrusSimulatorApp.scenarioExecution.scenarioMessages' | translate) }"
-                  >Show Messages</span
-                >
-              </button>
-              <button
-                type="button"
-                [routerLink]="['/scenario-parameter']"
-                [queryParams]="{ 'filter[scenarioExecutionId.in]': scenarioExecution.executionId }"
-                class="btn btn-info btn-sm"
-                data-cy="filterOtherEntityButton"
-              >
-                <fa-icon icon="gears"></fa-icon>
-                <span
-                  class="d-none d-md-inline"
-                  jhiTranslate="entity.action.show"
-                  [translateValues]="{ otherEntity: ('citrusSimulatorApp.scenarioExecution.scenarioParameters' | translate) }"
-                  >Show Parameters</span
-                >
-              </button>
-              <button
-                type="button"
-                [routerLink]="['/scenario-execution', scenarioExecution.executionId, 'view']"
-                class="btn btn-info btn-sm"
-                data-cy="entityDetailsButton"
-              >
-                <fa-icon icon="eye"></fa-icon>
-                <span class="d-none d-md-inline" jhiTranslate="entity.action.view">View</span>
-              </button>
-            </div>
-          </td>
+          <ng-container *ngIf="{ 'filter[scenarioExecutionId.in]': scenarioExecution.executionId } as queryParams">
```

Comment:
why that `if`? I think that wasn't here beforehand.

Reply:
ah, I see.. you wanted to use a variable for the `queryParams` later on. honestly, I think that's rather confusing. because it is not really an `if` condition. I would not recommend using it this way.

Reply:
yes it was created for reusing the same filter. I will change it back how it was